src/Idephix/Extension/Slack/Slack.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Idephix\Extension\Slack;
+
+use Idephix\IdephixInterface;
+use Idephix\Extension\IdephixAwareInterface;
+
+/**
+ * Description of Slack wrapper
+ *
+ * @author dymissy
+ */
+class Slack implements IdephixAwareInterface
+{
+    /**
+     * @var \Idephix\IdephixInterface
+     */
+    private $idx;
+
+    private $settings;
+
+    public function __construct($args = array())
```

Comment:
well yes, inside the sendToSlack method, using $idx->get('param') the extension can get every parameter it needs